edb/edgeql/parser/grammar/lexer.py

```diff
@@ -238,7 +238,7 @@ class EdgeQLLexer(lexer.Lexer):
 
         Rule(token='QIDENT',
              next_state=STATE_KEEP,
-             regexp=r'`([^`]|``)*`'),
+             regexp=r'`([^`]|``)+`'),
```

Comment:
I'm not sure I get these changes. Can you add a couple of syntax tests that wouldn't work with old regexes?

Reply:
The change is so that empty string is no longer a valid identifier. It seems like a kind of identifier that is likely to cause errors and confusion. Such as introspected name would be `"modname::"`, which although technically correct, looks more like some kind of error.

I can change it back if we really want to keep empty strings as valid names, I'm just not sure it's a good idea.

I can add some tests for empty string names.

Reply:
> The change is so that empty string is no longer a valid identifier. 

yeah, add a test for that!

Reply:
I've updated the tests and adjusted the regexp back to the original so that the error message about empty identifiers could be more specific.